android/src/main/java/expo/modules/blueskyvideo/FullscreenActivity.kt

```diff
@@ -25,40 +30,49 @@ class FullscreenActivity : AppCompatActivity() {
             return
         }
 
-        this.window.setFlags(
-            WindowManager.LayoutParams.FLAG_FULLSCREEN,
-            WindowManager.LayoutParams.FLAG_FULLSCREEN,
-        )
+        // Enable edge-to-edge mode but keep navigation bar persistent
+        WindowCompat.setDecorFitsSystemWindows(window, false)
 
-        val keepDisplayOn = this.getIntent().getBooleanExtra("keepDisplayOn", false)
-
-        if (keepDisplayOn) {
-            this.window.addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
+        val playerView = PlayerView(this).apply {
+            setBackgroundColor(Color.BLACK)
+            setShowSubtitleButton(true)
+            setShowNextButton(false)
+            setShowPreviousButton(false)
+            setFullscreenButtonClickListener {
+                finish()
+            }
+            layoutParams = ViewGroup.LayoutParams(
+                ViewGroup.LayoutParams.MATCH_PARENT,
+                ViewGroup.LayoutParams.MATCH_PARENT
+            )
+            useController = true
+            controllerAutoShow = false
+            controllerHideOnTouch = true
         }
+        playerView.player = player
+        setContentView(playerView)
 
-        // Update the player viewz
-        val playerView =
-            PlayerView(this).apply {
-                setBackgroundColor(Color.BLACK)
-                setShowSubtitleButton(true)
-                setShowNextButton(false)
-                setShowPreviousButton(false)
-                setFullscreenButtonClickListener {
-                    finish()
-                }
+        // Adjust layout for system insets to accommodate persistent nav bar
+        ViewCompat.setOnApplyWindowInsetsListener(playerView) { view, insets ->
+            val systemBarsInsets = insets.getInsets(WindowInsets.Type.systemBars())
+            view.setPadding(0, systemBarsInsets.top, 0, systemBarsInsets.bottom)
+            insets
+        }
 
-                layoutParams =
-                    ViewGroup.LayoutParams(
-                        ViewGroup.LayoutParams.MATCH_PARENT,
-                        ViewGroup.LayoutParams.MATCH_PARENT,
-                    )
-                useController = true
-                controllerAutoShow = false
-                controllerHideOnTouch = true
+        // Prevent hiding of the navigation bar
```

Comment:
Just curious why 🤔

Reply:
quite a few people have requested this, makes it easier to get out of the video since you can't swipe down to dismiss. i think that's the reasoning.